src/encord_active/lib/project/project.py

```diff
@@ -134,21 +135,46 @@ def __load_label_row_meta(self, subset_size: Optional[int]):
         label_row_meta_file_path = self.file_structure.label_row_meta
         if not label_row_meta_file_path.exists():
             raise FileNotFoundError(f"Expected file `label_row_meta.json` at {label_row_meta_file_path.parent}")
-        self.label_row_meta = {
+        self.label_row_metas = {
             lr_hash: LabelRowMetadata.from_dict(lr_meta)
             for lr_hash, lr_meta in itertools.islice(
                 json.loads(label_row_meta_file_path.read_text(encoding="utf-8")).items(), subset_size
             )
         }
 
     def __download_and_save_label_rows(self, encord_project: EncordProject):
-        label_rows = download_all_label_rows(encord_project, self.file_structure)
-        download_all_images(label_rows, self.file_structure)
+        label_rows = download_label_rows(encord_project, self.file_structure)
+        results = download_all_images(label_rows, self.file_structure)
+
+        lr_with_failed_images = {lr_hash for lr_hash, success in results.items() if success is False}
```

Comment:
Since Label row label hash can be `None` (shouldn't because it is filtered by we don't have a type that satisfies that) we [return](https://github.com/encord-team/encord-active/blob/31cb44182cf171418d51e8631ccf4f4054f654ce/src/encord_active/lib/project/project.py#L235-L236) `None` from the method if the is no labels hash. So in this case we are actually looking for explicit `False`.